### DISCUSSION TITLE
when will the new version be released?

### DISCUSSION THREAD

**Discussion by Sriram-L** [cpeditor/cpeditor#1042] [Development]
I have been waiting for the new version of cpeditor which has vim mode, syntax highlighting, and much more features promised earlier.
Also wanted to mention this, i love cpeditor! Very minimal, lightweight and simple to use. lately been practising cp only with this.

**Comment by coder3101** [comment]
It will be released once it is ready. For now, it is not even in master and no progress since last few months. Try `ksyntax-highlighting` branch and build with source.
